src/connector/src/with_options_test/mod.rs

```diff
@@ -70,19 +86,24 @@ pub fn update_with_options_yaml() -> String {
         for field in struct_item.fields {
             // Process each field
             if let Some(field_name) = &field.ident {
-                let serde_props = extract_serde_properties(&field);
+                let SerdeProperties {
+                    default_func,
+                    rename,
+                    alias,
+                } = extract_serde_properties(&field);
 
                 let field_type = field.ty;
                 let mut required = match extract_type_name(&field_type).as_str() {
                     // Fields of type Option<T> or HashMap<K, V> are always considered optional.
                     "HashMap" | "Option" => false,
                     _ => true,
                 };
-                let field_type = quote::quote!(#field_type).to_string();
+                let mut field_type = quote::quote!(#field_type).to_string();
```

Comment:
`to_string` does not look good as it'll insert spaces between tokens. What about `.to_token_stream().into_iter().join('')`?

Reply:
Weird
![Uploading image.png…]()